eng/build.yml

```diff
@@ -20,16 +20,17 @@ phases:
     name: ${{ parameters.agentOs }}
     ${{ if eq(parameters.agentOs, 'Windows_NT') }}:
       enableMicrobuild: true
-    enablePublishBuildArtifacts: true
+    enablePublishBuildArtifacts: false
     enablePublishBuildAssets: true
-    enablePublishTestResults: true
+    enablePublishTestResults: false
```

Comment:
nit: you can also just remove these "enable" parameters so nobody feels the urge to switch them to `true` in a bad merge or something.